src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml

```diff
@@ -3,9 +3,10 @@
 jobs:
 - job: Source_Build_Create_Tarball
   displayName: Source-Build Create Tarball
+  timeoutInMinutes: 180
```

Comment:
When you integrate with the CI changes I just merged, this timeout should be removed.  The tarball builds have been moved to a separate job now which have a 120 min timeout.

Reply:
Fixed.